test/integration/sanMvrvRatioTest.js

```diff
@@ -20,17 +22,16 @@ describe('SAN_MVRV_RATIO', () => {
   const response = san.SAN_MVRV_RATIO(slug, from, to)
```

Comment:
currency, timeBound?

Reply:
There should be also a test that ensures the old implementation works.
Example title: "old implementation works - w/o currency and timeBound"